### PR TITLE
Rework landing page appearance

### LINKED ISSUE #125: Style update to new flow
@jenn-rhim has some visual design updates for the new flow.

The updated assets should be added to this issue

### PR DESCRIPTION
**Pull request by lgebhardt**
Reworks the landing page appearance, per #125.

Two outstanding issues:
- The background graphic will need to be changed to replace the $
- The dashboard image needs to be remade at the larger size so it can responsively shrink if need be

Fix #125

### CODE REVIEW

**Comment by lgebhardt** [comment]
Full view (from Safari):
screen shot 2017-10-03 at 6 42 23 pm

Mobile view (from Chrome):
screen shot 2017-10-03 at 6 43 01 pm

The color difference is from the two different browsers.

File: app/assets/stylesheets/variables/fonts.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // .scss for multiline attributes
+@import url(http://fonts.googleapis.com/css?family=Open+Sans);
```
Comment (ayumi) [inline app/assets/stylesheets/variables/fonts.scss]: If this is necessary, we should self host it and use HTTPS.

File: config/locales/en.yml [hunk 2]
```diff
@@ -301,15 +301,17 @@ en:
...
+    landing_header: "Brave Payments"
     landing_body_html: |
-      <p>Users do not want ads, but you still have bills to pay. Brave Bounty streams micropayments from the audience
-      to you.</p>
-      <p>Once full verified you will begin to receive daily micropayments from users in your BAT wallet.</p>
-    landing_new_header: "New to Brave Bounty?"
-    landing_new_body: "Start the process of registering your domain by verifying your email address."
+      <p>Our users do not want ads, but you still have bills to pay. Brave Payments streams micropayments from
+       the audience to you.</p>
+      <p>Once full verified, you will begin to receive daily micropayments from users in your BAT wallet.</p>
```
Comment (ayumi) [inline config/locales/en.yml]: s/full/fully

File: app/assets/stylesheets/variables/fonts.scss (hunk 1 above)
Comment (lgebhardt) [reply app/assets/stylesheets/variables/fonts.scss]: I can test again, but without it I was getting the browser falling back to the base sans-serif font.

**Comment by lgebhardt** [comment]
@ayumi Thanks for the fast review. Can you clarify "On my Mac the page is 100px too far down the page (still above the fold but just barely)"? Do you want less blank space above and below the logo?

**Comment by lgebhardt** [comment]
Spelling and https issues fixed.

**Comment by ayumi** [comment]
Sure – again it's very minor (50–100px off) – on my MBP 13 in Chrome it looks like this:
screen shot 2017-10-03 at 23 11 58

Comparison with Medium.com:
screen shot 2017-10-03 at 23 13 13

**Comment by lgebhardt** [comment]
I agree, I was matching the layout in the writeup and forgot to ask about the top spacing. I can tweak that first thing in the morning.

**Pull request merged**